src/lib/homestore_backend/tests/hs_repl_test_helper.hpp

```diff
@@ -56,6 +57,7 @@ class HSReplTestHelper {
                 sync_point_num_ = sync_point;
                 homeobject_replica_count_ = 0;
                 uint64_id_ = INVALID_UINT64_ID;
+                v_chunk_id_ = INVALID_CHUNK_NUM;
```

Comment:
NIT: I prefer to  cast  homestore::chunk_num_t(uinit_16) to uint_64 here, and then cast it back.

Reply:
I'll merge first, and if necessary,  I will refine in next pr.
BTW, what are the pros in this way? 😃

Reply:
hs_repl_test_helper.hpp  is the framework, it does not know what is a shard_id_t , pg_id_t , blob_id_t and chunk_id_t. these types are only know application layer.

if we add v_chunk_id here, should we also add  shard_id_t, pg_id_t and others? 

so I think we can use this 64bit as a common shared data across different processed, and application layer should translate this to what it wants